app-text/xournal/xournal-0.4.8.2016-r1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+[[ ${PV} == *9999 ]] && GNOME2_EAUTORECONF=yes
+
+inherit gnome2
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.code.sf.net/p/xournal/code"
+	unset SRC_URI
+else
+	KEYWORDS="amd64 ~ppc64 ~x86"
```

Comment:
One nit: ekeyword ~all in future (don't commit straight-to-stable) usually, although we've done it for some Clang 16 fixes

Reply:
Oops missed that line.